test/testlib/nuodb_admin_utilities.go

```diff
@@ -303,3 +303,15 @@ func AwaitServerState(t *testing.T, namespace string, adminPod string,
 		return false
 	}, timeout)
 }
+
+// VerifyLabels checks if adminPod in namespace has all the label values in expectedLabelValues
+func VerifyLabels(t *testing.T, namespace string, adminPod string, expectedLabelValues map[string][]string) {
+	admins, err := GetDomainServersE(t, namespace, adminPod)
+	require.NoError(t, err)
+	require.Contains(t, admins, adminPod)
```

Comment:
Are we comparing different types here?

Reply:
I am not sure what you mean.

Reply:
`admins` is `map[string]NuoDBServer` and `adminPod` is `string`. Does `require.Contains` magically search the expected string in the map keys?

Reply:
It does. From the [docs](https://pkg.go.dev/github.com/stretchr/testify@v1.8.4/require#Contains):
`assert.Contains(t, {"Hello": "World"}, "Hello")`

Reply:
It is iterating over all map keys: https://github.com/stretchr/testify/blob/b074924938f86d417f1c9a845c7e8b0784d7f937/assert/assertions.go#L904